packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -799,53 +804,74 @@ export const sendGsheetsNotification = async (
             targetType: 'gsheets',
             status: SchedulerJobStatus.STARTED,
```

Comment:
this is missing an `await` is this intended?

Reply:
actually in my editor I see a lot of warnings about missing awaits in this whole file

Reply:
Updated and added await for all log methods in this file . Good catch. I didn't get any warnings, I'm wondering if you have an extension I don't...